### PR TITLE
Feature/segun/user profile

### PR DESCRIPTION
**Pull request by Segun-Ade**
#### What does this PR achieve?
This PR implements the user profile view

### CODE REVIEW

File: src/components/userDashboard/components/profile/api/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    const token = await auth.currentUser.getIdToken();
+    const config = {
+        headers: {
+            'Content-Type': 'application/json',
+            authorization: `Bearer ${token}`,
+        },
+    };
+    const res = await axios.put(API_URL, info, config);
+    return res.data;
+};
```
Comment (enyeInc) [inline src/components/userDashboard/components/profile/api/index.js]: where is this function being used?

File: src/components/userDashboard/components/profile/components/AddressForm.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        e.preventDefault();
+    };
+
+    return (
+        <Form layout="inline" onSubmit={handleSubmit} className="login-form">
+            <Form.Item>
+                <Button type="primary" htmlType="submit" className="login-form-button">
+                    {current ? UPDATE : ADD}
+                </Button>
+
```
Comment (enyeInc) [inline src/components/userDashboard/components/profile/components/AddressForm.jsx]: delete unused line

File: src/components/userDashboard/components/profile/components/AddressTable.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Table
+} from 'antd';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+function buttonActions() {
+    return (
+        <div />
+    );
+}
```
Comment (enyeInc) [inline src/components/userDashboard/components/profile/components/AddressTable.jsx]: remove this function

File: src/components/userDashboard/components/profile/api/index.js (hunk 1 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/profile/api/index.js]: I'll move it to sagas now

File: src/components/userDashboard/components/profile/components/AddressForm.jsx (hunk 2 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/profile/components/AddressForm.jsx]: I'll do that

File: src/components/userDashboard/components/profile/components/AddressTable.jsx (hunk 3 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/profile/components/AddressTable.jsx]: I'll  update the function

**Pull request merged**